Adding CONTRIBUTING.md file. (#14)
redhat-beyond/ResuMe: CONTRIBUTING.md

# CONTRIBUTION
## Codding standard
We expect all Python code to conform to the [PEP8](https://www.python.org/dev/peps/pep-0008) standard.
## Pull Requests(PR)
 - Each Pull Request should focus on single repository principle.
 - **The Pull Request should not break any of the existing functionality**
 - <ins>Description-</ins> the PR description should explain what changes have been made, and why. In addition, they should include a link to the relevant `issue`
 - Each PR should include 1-5 `commits`
 - Make sure the PR passes CI checks.

>  - Each pull request requires approves of at least <ins>2 team members</ins> before merge.

Reviewer: ```suggestion
 - Each pull request requires the approval of at least <ins>2 team members</ins> before merging.
```

>  - Each pull request requires approves of at least <ins>2 team members</ins> before merge.

Reviewer 2: How about this?
 Each pull request requires approves of at least <ins>2 team members</ins> before merge.
 The team members of ResuMe are:
- Omer Shor (github user
- Yuli (github user)
- Matan  (github user)
- etc

## Getting started
### Prerequisites
 - Install [Vagrant](https://www.vagrantup.com/downloads).
 - Install [Virtual Box](https://www.virtualbox.org/wiki/Downloads).
### Initializing virtual environment 

> 1. Clone the [ResuMe](https://github.com/redhat-beyond/ResuMe) repository Using the command `git clone https://github.com/redhat-beyond/ResuMe.git`.

Reviewer: You need to create a fork and a separate branch for development.

# CONTRIBUTING
When contributing to this repository, please first discuss the change you wish to make via issue, GitHub, or any other method with the owners of this repository before making a change.

> Please note we have a code of conduct, please follow it in all your interactions with the project.

Reviewer 3: ```suggestion
Please follow our code of conduct in all your interactions with the project.
```
